### DISCUSSION TITLE
allow attaching bzip/xz/lzma tarballs

### DISCUSSION THREAD

**Discussion by evgeni** [community/community#11294] [Projects and Issues]
Ohai,

currently, if one wants to attach a compressed file, it is only possible to do so as Zip of GZip (see https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/attaching-files), it'd be nice if we could also get more modern compression algorithms (especially XZ, which almost everyone is moving to).

Thanks!

**Comment by nodeg** [comment]
`.txz` would also be very nice, since SUSE uses it for compressing their [supportconfig](https://documentation.suse.com/sles/15-SP4/html/SLES-all/cha-adm-support.html).

**Comment by andreldmonteiro** [comment]
Support for xz should had already be implemented, its basic, really.
